### ISSUE TITLE
Pls pls pls, make the current working directory path selectable. Any displayed text in the WARP terminal UI should be selectable (and copyable))

### ISSUE THREAD

**Issue by razine-bensari** [warpdotdev/Warp#2936]
### Describe the solution you'd like?

```text
Anything that is displayed in the WARP terminal UI should be selectable. Everything. All text displayed should at the very least be selectable so I can copy it and paste it somewhere
```

### Is your feature request related to a problem? Please describe.

```text
I am always frustrated when I am using git in the WARP terminal and I can see the branch name in the directory path and I switch branches but then I cannot copy paste the path. This is a simple use case but there are plenty use cases where being able to select and copy what is displayed in the WARP terminal UI can be useful
```

### Additional context

I love u

### How important is this feature to you?

4

### Warp Internal (ignore) - linear-label:770f6576-d6c0-4e4f-a259-fc64b5156087

None

**Comment by dannyneira** [comment]
Hi @razine-bensari ,

I understand the need to be able to copy all the text. Given how the prompt/input editor are separate in Warp ( this is what allows for IDE-like editing for the input ), we've added some options in the right-click menu of the prompt to help with this use case. Please see our docs for more info https://docs.warp.dev/features/prompt#how-to-access-it

TL;DR of the options are in my screenshot below.

![CleanShot 2023-04-17 at 14 17 52](https://user-images.githubusercontent.com/16809145/232600658-acb6725c-b662-44fb-a5ce-2d823f7a0667.png)

That being said, I think we are eventually working towards most things that make sense being selectable/copyable/etc in Warps UI. Closing as completed since an option for the use case exists.

**Comment by scottmcpherson** [comment]
I'm glad there's a workaround, but I never would've guessed that. It could be more intuitive. I suspect many people, including myself, are coming from iTerm2 with a typical setup where you can copy the branch name directly from the terminal output by selecting it with the cursor and cmd+c copying it.

**Comment by razine-bensari** [comment]
Amazing! Thank you very much ❤️
